docs/content/posts/2024-07-15-release-1.20.0.md

```diff
@@ -0,0 +1,560 @@
+---
+authors:
+- reswqa:
+  name: "Weijie Guo"
+  twitter: "WeijieGuo12"
+- 1996fanrui:
+  name: "Rui Fan"
+  twitter: "1996fanrui"
+
+date: "2024-07-15T08:00:00Z"
+subtitle: ""
+title: Announcing the Release of Apache Flink 1.20
+aliases:
+- /news/2024/07/15/release-1.20.0.html
+---
+
+The Apache Flink PMC is pleased to announce the release of Apache Flink 1.20.0. As usual, we are
+looking at a packed release with a wide variety of improvements and new features. Overall, 142
+people contributed to this release completing 13 FLIPs and 300+ issues. Thank you!
+
+Let's dive into the highlights.
+
+# Standing on the Eve of Apache Flink 2.0
+
+Flink 1.0 was released seven years ago. Since several months, the community is actively planning and taking steps towards
+the next major release. The new 1.20 release is planned to be the last minor release before Flink 2.0, which is anticipated by the end of 2024.
+
+Start from Flink 1.19, the community has decided to officially deprecate multiple APIs that were approaching 
```

Comment:
this sentence does not read well. How about something like "At Flink 1.19, the community decided to deprecate multiple APIs in preparation for Flink version 2."